flask_admin/helpers.py

```diff
@@ -132,7 +132,8 @@ def prettify_class_name(name):
 
 def is_safe_url(target):
     # prevent urls starting with "javascript:"
-    target = target.strip()
+    _substitute_whitespace = compile(r'[\s\x00-\x08\x0B\x0C\x0E-\x19]+').sub
```

Comment:
`_substitute_whitespace` is better to be a const.

Reply:
Agree here - please change it to const and I'll merge ASAP.